src/test/java/cz/codecamp/logger/FileHandlerTest.java

```diff
@@ -0,0 +1,54 @@
+package cz.codecamp.logger;
+
+import cz.codecamp.logger.fileHandlers.FileHandler;
+import cz.codecamp.logger.fileHandlers.IntervalBasedFileHandler;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Filip Ocelka on 13.10.2016, filip.ocelka@gmail.com
+ */
+public class FileHandlerTest {
+
+    private FileHandler fileHandler;
+    private DateTimeFormatter formatter;
+    private LocalDateTime dateTimeNow;
+
+    @Before
+    public void setUp() throws Exception {
+        fileHandler = new IntervalBasedFileHandler("testLogFile.log", ChronoUnit.DAYS,1);
+        formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy-HH-mm-ss");
+        dateTimeNow = LocalDateTime.now();
+    }
+
+    @Rule
+    public TemporaryFolder tempFolder = new TemporaryFolder();
```

Comment:
👍 super použití `TemporaryFolder`... hm, tak výraz folder používá krom windowsů ještě jUnit 😄